.travis.yml

```diff
@@ -1,31 +1,63 @@
 language: ruby
+
 sudo: false
 dist: trusty
+
 rvm:
   - 2.3.0
+
+# Allow Emacs snapshot builds to fail and don't wait for it
 matrix:
   fast_finish: true
   allow_failures:
     - env: EMACS_VERSION=snapshot
+
+# Cache stable Emacs binaries and packages
 cache:
-  - bundler
-  - apt: true
-  - directories:
+  bundler: true
+  apt: true
+  timeout: 691200
+  directories:
       - "$HOME/emacs"
+      - "$HOME/.emacs.d"
```

Comment:
Why are we caching `.emacs.d` exactly?  We never install ELPA packages in this job, right?